### PR TITLE
feat(CFW): refactor the query logic for address group member resource

### PR DESCRIPTION
**Pull request by profoundwu**
**What this PR does / why we need it**:
 The creation interface supports filling in the address range, but the query interface does not support filtering by address range. So change the query logic from filtering by address to filtering by id for address group member resource.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1.  change the query logic from filtering by address to filtering by id for address group member resource
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/cfw" TESTARGS="-run TestAccAddressGroupMember_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cfw -v -run TestAccAddressGroupMember_basic -timeout 360m -parallel 4
=== RUN   TestAccAddressGroupMember_basic
=== PAUSE TestAccAddressGroupMember_basic
=== CONT  TestAccAddressGroupMember_basic
--- PASS: TestAccAddressGroupMember_basic (25.15s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cfw       25.205s

make testacc TEST="./huaweicloud/services/acceptance/cfw" TESTARGS="-run TestAccAddressGroupMember_addressRange"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cfw -v -run TestAccAddressGroupMember_addressRange -timeout 360m -parallel 4
=== RUN   TestAccAddressGroupMember_addressRange
=== PAUSE TestAccAddressGroupMember_addressRange
=== CONT  TestAccAddressGroupMember_addressRange
--- PASS: TestAccAddressGroupMember_addressRange (23.23s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cfw       23.300s
```

### CODE REVIEW

File: huaweicloud/services/cfw/resource_huaweicloud_cfw_address_group_member.go [hunk 1]
```diff
@@ -147,63 +147,71 @@ func resourceAddressGroupMemberRead(_ context.Context, d *schema.ResourceData, m
...
 
-func buildGetAddressGroupMemberQueryParams(d *schema.ResourceData) string {
-	res := "?offset=0&limit=10"
-
-	res = fmt.Sprintf("%s&set_id=%v", res, d.Get("group_id"))
-
-	res = fmt.Sprintf("%s&address=%v", res, d.Get("address"))
-
-	return res
+func readAddressGroupMembers(setID string, client *golangsdk.ServiceClient) ([]interface{}, interface{}, error) {
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cfw/resource_huaweicloud_cfw_address_group_member.go]: you can use this func in test

Comment (profoundwu) [reply huaweicloud/services/cfw/resource_huaweicloud_cfw_address_group_member.go]: fixed

**Pull request merged**